CHANGELOG.MD

```diff
@@ -1,3 +1,21 @@
+#### v2.1.0
+
+- performance testing (browserMob and TTFB timings)
+- improvements for clicking element, every step waits for visibilityOf element
+- added wait method to helpers
```

Comment:
This method is also exported outside Kakunin, so it would be nice if we add here an information.